docs/sources/developer/architecture/components/api.md

```diff
@@ -0,0 +1,78 @@
+# API
+
+The Faro API is the primary interface for end-users to interact with Faro. It is the central collector of signals, and
+also exposes helper methods for extensions such as metas.
```

Comment:
Maybe do a brief description of 'signals' stands for

Reply:
This should be part of another set of docs as this set refers only to the internal architecture of the core package.